#1 Fixed setting of boolean flag that toggles whether a full feature …

AlexanderSchultheiss opened the pull request:
…model extraction is started. The flag is now determined based on the comparison to `FullExtraction.class.getName()` (instead of a String literal).

AlexanderSchultheiss committed:
#1 Fixed setting of boolean flag that toggles whether a full feature model extraction is started. The flag is now determined based on the comparison to `FullExtraction.class.getName()` (instead of a String literal).

AlexanderSchultheiss committed:
Updated version number

theMichaelScheurer reviewed (changes requested):
Changing the version number seems to have broken the extraction script:
```
...
Executing variability extraction of BusyBox.
Error: Unable to access jarfile Extraction-1.0.0-jar-with-dependencies.jar
Done.
```
(./docker-resources/extract.sh expecting excact version number?)

Extraction did work for commit f4c1423

AlexanderSchultheiss committed:
Changed Maven's configuration to omit the version number from the jar file, and I updated the configuration files accordingly

AlexanderSchultheiss commented:
The moment when you are sure that a version change will not break anything and do not test the build.... 
I changed Maven's configuration to omit the version number from the jar file, and I updated the configuration files accordingly.
